src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const TOTAL_PRICE = PRICE_PER_DAY * days;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  if (days < SHORT_TERM) {
+    return TOTAL_PRICE;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return TOTAL_PRICE - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
Instead of combining the conditions, you can just firstly check if it's a long term, then if it's a short term.